src/lib/accounts/accounts.ts

```diff
@@ -2,7 +2,7 @@ import axios, { AxiosRequestConfig, AxiosResponse } from 'axios';
 import moment from 'moment';
 import { BaseLogger } from 'pino';
 
-import {Intercept} from '../axios-logger/axios';
+import {intercept} from '../axios-logger/axios';
```

Comment:
The imported function is imported on `src/lib/axios-logger/axios.ts#L49` as `Intercept`